domains/lumi.json

```diff
@@ -6,5 +6,6 @@
   },
   "record": {
     "A": ["76.76.21.21"]
+    "TXT" : ["dh=e00668e963ac416881a25e0bb5f87429a5eb0792"]
```

Comment:
Wrong way to use discord TXT verification, please go here to see how to do it: https://is-a.dev/docs/discord-verification/

Reply:
Btw the actual problem was because you didn't add a comma at the end of the A line

Reply:
I actually completely over read that I need to create a new file. Sorry!